custodial-copy-confirmer/src/test/scala/uk/gov/nationalarchives/confirmer/MainTest.scala

```diff
@@ -14,24 +14,24 @@ class MainTest extends AnyFlatSpec {
     val existingRef = UUID.randomUUID
```

Comment:
I couldn't comment on line 13, so I'll do it here. Shouldn't this now be
"write to dynamo and delete the messages from the queue only if the object exists" or something like that?